test/node-rpc-test.js

```diff
@@ -425,7 +425,7 @@ describe('RPC', function() {
       const {genesis} = node.network;
       let entry = await node.chain.getEntry(genesis.hash.reverse());
 
-     // Get current chain tip and chain height
+    // Get current chain tip and chain height
```

Comment:
If you're going to touch this line, it should be like this:

```js
      // Get current chain tip and chain height
      const chainHeight = node.chain.tip.height + 1;
      const chainTip = util.revHex(node.chain.tip.hash);
```

Reply:
Fixed this annoying formatting change. I did not intend to touch these lines.